src/components/ArticleCard/ArticleCard.spec.js

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import ArticleCard from './index';
+
+describe('ArticleCard', () => {
+  let articleCard;
+  const title = 'Pointing';
+  const description =
+    'Things are going to be fine. All you need to do is stay focused and believe in yourself';
+  const body =
+    'The journey to become world class may be scary and daunting. All you need to do is stay focused and believe in yourself';
+  const image = 'image';
+  const authorImage = 'author-image';
+  const createdAt = 'June 25';
+  const author = 'Mark';
+  const readTime = '2 mins read';
+
+  beforeEach(() => {
+    articleCard = mount(
+      <BrowserRouter>
+        <ArticleCard
+          title={title}
+          description={description}
+          body={body}
+          image={image}
+          author-image={authorImage}
+          createdAt={createdAt}
+          author={author}
+          readTime={readTime}
+        />
+      </BrowserRouter>
+    );
+  });
+
+  it('renders the article card successfully', () => {
+    expect(articleCard).toBeDefined();
+    expect(articleCard.find('div').length).toBe(4);
+    expect(articleCard.find('img').length).toBe(2);
+    expect(articleCard.find('h1').length).toBe(1);
+    expect(articleCard.find('p').length).toBe(3);
+    expect(articleCard.find('span').length).toBe(3);
+    expect(articleCard.find('button').length).toBe(1);
+  });
+
+  it('should render an article card with a title', () => {
+    articleCard = mount(
+      <BrowserRouter>
+        <ArticleCard
+          title={title}
+          description={description}
+          body={body}
```

Comment:
@  I noticed you are doing different assertions on the aritcleCard properties, which I feel could be done in one assertion. Just use a generic assertion It() title and you will be fine.

Reply:
Thanks a lot, @kechyy. 👍 
Please, can you explain what you meant by a `generic assertion`?